gradio/theming/utils.py

```diff
@@ -0,0 +1,18 @@
+class Theme:
```

Comment:
I mentioned this in #3008 but I think a `to_json` method will help users create new themes from themes uploaded to the hub